enhancements/rhcos/static-networking-enhancements.md

```diff
@@ -0,0 +1,104 @@
+
+### Story 2
+A customer has a fleet of bare metal machines they want to provision with RHCOS and static networking, that will later be used for the installation of OpenShift Container Platform (OCP).  They use some automation to generate unique Ignition configs that include the static networking configuration.  Using similar automation, they use `coreos-installer iso embed` to embed the Ignition configs into unique ISOs per machine.  These ISOs can then be attached to the baseboard machine controller (BMC) or lights out management (LOM) of the host, at which point the hosts can be booted and RHCOS will be automatically installed with the static networking configured and Ignition config applied.
+
+### Story 3
+A customer wants to deploy OCP onto VMware vSphere and use static networking for their hosts.  For their specific environment, it is impractical for them to use the bare metal ISO install path.  During the definition of the VM using the OVA image, the customer provides data to a `guestinfo` property that specifies the static networking configuration in the form of `dracut` kernel arguments.  When the VM is booted, the values in the `guestinfo` property are applied before networking is brought online and the VM continues to boot normally.
+
+### Implementation Details/Notes/Constraints
+This enhancement is being delivered across multiple projects and requires coordination among all of them.  This requires changes to at least:
+  - `coreos-installer`
+  - `ignition-dracut`
+  - `coreos-assembler`
+  - `afterburn`
+
+### Risks and Mitigations
+Landing the changes first in FCOS and then delivering them in RHCOS has been the model we've tried to ascribe to as much as possible.  However, there are often challenges in enabling the changes in RHCOS due to differences in things like software versions and kernel support.
+
+If these changes are unable to be made to RHCOS, we may need to consider shipping newer versions of certain pieces of software in RHCOS that is not available in RHEL.
+
+The biggest risk to delivering the enhancement to RHCOS is the ability to generate and deliver the live ISO (see [#210](https://github.com/openshift/enhancements/pull/210)).
+
+At the very worst, we would continue to ship the legacy `coreos-installer` and artifacts for RHCOS, which would not contain any of the enhancements outlined here.
+
+### Upgrades
+These enhancements should only affect a subset of initial FCOS/RHCOS installs.  The upgrade path should not be affected.
+
+Any hosts that have been installed with static networking configured should be able to upgrade successfully and maintain their networking configuration.
+
+### Test Plan
+  - Boot FCOS/RHCOS live ISO and confirm user is dropped into a live environment
+    - Confirm the same with networking enabled/disabed
```

Comment:
This requirement is more environmental rather than software configuration. How about "Confirm the same experience on a network with DHCP and without DHCP".

Reply:
i.e. before this change on a network without DHCP you end up at an error prompt in the initrd.